src/main/java/de/teamlapen/vampirism/tileentity/TotemTileEntity.java

```diff
@@ -450,19 +462,20 @@ public void tick() {
                         IFaction faction = VampirismAPI.factionRegistry().getFaction(entity);
                         if (faction == null) continue;
                         if (entity instanceof ICaptureIgnore) continue;
+                        CaptureInfo captureInfo = new CaptureInfo(this);
```

Comment:
Does this have to be created for every single entity in the area or can we use a single instance for all?